lib/ui/painting.dart

```diff
@@ -1418,6 +1418,44 @@ class Canvas extends NativeFieldWrapperClass2 {
   /// entire group can be made transparent using the [saveLayer]'s paint.
   ///
   /// Call [restore] to pop the save stack and apply the paint to the group.
+  ///
+  /// ## Using saveLayer with clips
+  ///
+  /// When a rectanglular clip operation (from [clipRect]) is not axis-aligned
+  /// with the display, or when the clip operation is not rectalinear (e.g.
```

Comment:
> with the display

Technically with the raster buffer, which isn't necessarily the screen if there's a rotation or perspective transform between the current raster buffer and the screen.